infra.bs

```diff
@@ -258,11 +262,44 @@ inclusive.
 
 <h3 id=strings>Strings</h3>
 
-<p>A <dfn export>string</dfn> is a sequence of <a>code points</a>. Strings are denoted by double
-quotes and monospace font.
+<p>A <dfn export>JavaScript string</dfn> is a sequence of unsigned 16-bit integers, also known as
+<dfn export lt="code unit">code units</dfn>.
+
+<p class=note>This is different from how the Unicode Standard defines "code unit". In particular it
+refers exclusively to how the Unicode Standard defines it for Unicode 16-bit strings. [[UNICODE]]
+
+<p>A <a>JavaScript string</a> can also be interpreted as containing <a>code points</a>, per the
+conversion defined in <a>The String Type</a> section of the JavaScript specification. [[!ECMA-262]]
+
+<p class=note>This conversion process converts surrogate pairs into their corresponding
+<a>scalar value</a> and maps isolated surrogates to their corresponding <a>code point</a>, leaving
+them effectively as-is.
```

Comment:
I would be very interested in seeing an example of this conversion in action, that illustrates both of these points.